preprocessing/nextclade/src/loculus_preprocessing/nextclade.py

```diff
@@ -253,66 +357,27 @@ def enrich_with_nextclade(  # noqa: C901, PLR0912, PLR0914, PLR0915
     """
     unaligned_nucleotide_sequences: dict[
         AccessionVersion, dict[SegmentName, NucleotideSequence | None]
-    ] = {}
-    alerts: Alerts = Alerts()
-    input_metadata: dict[AccessionVersion, dict[str, Any]] = {}
-    aligned_aminoacid_sequences: dict[
-        AccessionVersion, dict[GeneName, AminoAcidSequence | None]
-    ] = {}
+    ] = defaultdict(dict)
     aligned_nucleotide_sequences: dict[
         AccessionVersion, dict[SegmentName, NucleotideSequence | None]
-    ] = {}
+    ] = defaultdict(dict)
+    alerts: Alerts = Alerts()
+    input_metadata: dict[AccessionVersion, dict[str, Any]] = {}
     for entry in unprocessed:
         id = entry.accessionVersion
         input_metadata[id] = entry.data.metadata
         input_metadata[id]["submitter"] = entry.data.submitter
         input_metadata[id]["group_id"] = entry.data.group_id
         input_metadata[id]["submittedAt"] = entry.data.submittedAt
-        aligned_aminoacid_sequences[id] = {}
-        unaligned_nucleotide_sequences[id] = {}
-        aligned_nucleotide_sequences[id] = {}
-        alerts.warnings[id] = []
-        alerts.errors[id] = []
-        num_valid_segments = 0
-        num_duplicate_segments = 0
```

Comment:
moved to `add_segment_name_error_warning` - will be replaced with segment assignment functions in future